src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/220932893-c9bdda42-d25d-4627-84dc-a190fd104dd7.png)

Reply:
After removing static from Color getRandomColor() method the program does not want to compile and show (non-static method getRandomColor() cannot be referenced from a static context) in Main method (Ball ball1 = Lottery.getRandomBall(); ) 
I don't know how to fix that